### PR TITLE
feat(Select): show tooltip if option label overflows #minor

### PR DESCRIPTION
**Pull request by rayhan-ab**
in old select component, there should be tooltip for the truncated option.
image

simple solution: use Select from global-ui instead of ui-lib, but it doesn't solve the issue that happens in other places that still use ui-lib's select

### CODE REVIEW

**Comment by rayhan-ab** [comment]
expected result
image

File: src/components/Select/index.tsx [hunk 1]
```diff
@@ -25,6 +34,7 @@ export const Select = ({ isMulti = false, onChange, dataQa = null, dataQaProps =
         noOptionsMessage={noOptionsMessage}
         components={{
           DropdownIndicator,
+          Option: !props.formatOptionLabel ? Option : components.Option,
```
Comment (rayhan-ab) [inline src/components/Select/index.tsx:37]: if the component use `formatOptionLabel`, then `formatOptionLabel` should be the one responsible for displaying the tooltip

File: src/utils/dom.tsx [hunk 2]
```diff
@@ -7,41 +7,49 @@
...
   if (!refDom) {
-    return 0
+    return 0;
   }
 
-  const computedStyle = window.getComputedStyle(refDom)
-  return parseInt(computedStyle.paddingRight || '0', 10)
-}
+  const computedStyle = window.getComputedStyle(refDom);
+  return parseInt(computedStyle.paddingRight || "0", 10);
```
Comment (rayhan-ab) [inline src/utils/dom.tsx:42]: perhaps we want to update prettier to match AP later. cc @aris-fm @yanuaracb

**Review by castrix — APPROVED** [review]
IIRC @yanuaracb mentioned that we will be leaning to use ui-lib instead of global-ui, so I think this is a good solution

File: src/components/Select/index.tsx (hunk 1 above)
Comment (aziz-zen-ab) [reply src/components/Select/index.tsx:37]: Does this component included in the docs? or if it is not yet, you can put this in to a comment

Comment (rayhan-ab) [reply src/components/Select/index.tsx:37]: what do you mean? sorry, i don't catch it

**Pull request merged**